styles.css

```diff
@@ -3,53 +3,122 @@
   height: 100vh;
   width: 100vw;
   display: grid;
-  grid-template:
-    'header main main main main'
-    'header main main main main'
-    'header section section section section'
-    'header section section section section'
-    'header section section section section'
-    
+  grid-template-columns: 20% 80%;
+  /* grid-template-rows: 65% 35%; */
+
 }
 
 header {
-  display: grid;
-  grid-row-start: span 5;
-  border: solid;
+  margin-left: 2%;
 
 }
 
 main {
   display: grid;
   grid-row: 1 / span 2;
   grid-column: 2 / span 4;
+  /* grid-auto-rows: 35%; */
 
 }
 
 section {
   display: grid;
   grid-row: 3 / span 3;
   grid-column: 2 / span 4;
+  border: solid;
+  /* grid-auto-rows: 65%; */
 }
 
+article{
+  height: 30vh;
+  width: 25vw;
+  border: solid;
+}
+
+/* Add div class name for styling/color */
+div{
+  border: solid;
+}
 
-.idea-images,
-.search-image {
+nav{
+  grid-template-rows: 30% 70%;
+  border-top: solid;
+  border-bottom: solid;
+  font-size: 10px;
+  /* align-content: center; */
+}
+
+.starred-button {
+  width: 175px;
+  margin-bottom: 5%;
+  margin-top: 5%;
+  margin-left: 2%;
+  margin-right: 2%;
+}
+
+.idea-images {
   height: 5%;
   width: 5%;
   background-color: #1F1F3D;
 }
 
+.search-image{
+  height: 100%;
+  width: 100%;
+  background-color: #353567;
+}
 
+.title,
+.body{
+  display: grid;
+  align-content: start;
+}
+
+.title {
+  height: 70px;
+  grid-template-rows: 30% 70%;
+}
+
+.body {
+  height: 120px;
+  grid-template-rows: 20% 80%;
+}
+
+.save-button {
+  height: 45px;
+}
 
 form {
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  align-items: center;
   justify-content: center;
   border: solid;
 }
-/*
-section {
-  display: flex;
+
+.search{
+  display: grid;
+  grid-template-columns: 10% 90%;
+}
+
+.search>button{
+  /* height: 100;
+  width: 10%; */
+  /* align-items: center; */
+  background-color: #353567;
+}
+
+.search>.text {
+  /* height: 30px; */
+  /* width: 90%; */
+  align-items: center;
   justify-content: flex-end;
```

Comment:
It seems as if this property was left over from when we were working with flex box instead of grid.